LICENSE

```diff
@@ -1,6 +1,14 @@
+The contained files are modified from CellProfiler/python-bioformats. The
+copied files were under BSD license, with the exception of the resource files
+within 'tests', which are unmodified and licensed under the GNU General
```

Comment:
Then maybe this project should be GPL

Reply:
The omexml and test file (not resourcess) are under BSD. Under Bio-formats it says:

> CellProfiler is licensed under the GNU General Public License version
> 2.  See the file COPYING for the terms of the GNU GPL license.
> 
> Certain files are licensed under the more permissive BSD license.
> This is indicated near the top of the relevant files.  The BSD license
> is as follows:

and the omexml and omexml test files are BSD, as stated at the top of each.

Reply:
It's not good practice to mix licences as this will affect their distribution. You either stick with GPL or strip out files that require it.

Reply:
Well... I could remove the tests for now, with the intention to make my own test files and add them back in later, which leaves this BSD-3. Does that seem sensible?

Reply:
I've done that now but I can easily change it back